### ISSUE TITLE
Objective

### ISSUE THREAD

**Issue by carodelafuente** [CrystalCo/bwa-u4-feature-request#1]
This feature might be a bit too simple. I think you should find some ways to make it a bit more challenging for yourself. Maybe implement a whole new functionality on your app?

For example: implementing an audio preview for each song.

**Comment by CrystalCo** [comment]
I agree.  I would have loved to spend more time on something more challenging.  Unfortunately, there was a bug in my Jammming project which prevented my app from working properly.  I asked for help and a codeacademy rep had to escalate the issue to their team of engineers. They said to wait a day or two for their response, but I never heard back.  Not only did this delay my time from working on the feature request, but I couldn't test out any of the more challenging features I had in mind.  An advisor said I didn't need the app to work in order to finish the feature request portion, however, how would I know if the feature request would work if we couldn't even figure out what was wrong with my code?  Without knowing what part of my code I had messed up, I had a lot of doubt about whether I understood the material correctly.  I think the issue lied somewhere in the Spotify.js code, because I wasn't getting a token and thus none of the functions were working.

In conclusion, being strapped on time and most of the advanced features requiring an access token led me to choose the first recommendation provided on the assignment page. Apart from that issue, I really appreciate the feedback :) Thank you
